src/main/java/com/exactpro/th2/conn/MicroserviceMain.java

```diff
@@ -162,7 +163,12 @@ public static void main(String[] args) {
                     EventType.SERVICE_EVENT, serviceEventsRoot.getId()
```

Comment:
Root event is built without the book name from the common factory

Reply:
I think we should add `bookName` to `com.exactpro.th2.common.event.Event` and to `com.exactpro.th2.common.event.EventUtils#toEventID`.
I'll do it.

Reply:
done